jwk_client.go

```diff
@@ -38,25 +37,25 @@ func NewJWKClient(options JWKClientOptions, extractor RequestTokenExtractor) *JW
 		extractor = RequestTokenExtractorFunc(FromHeader)
```

Comment:
Can you also add a comment to `NewJWKClient` regarding how the caching works by default.